lifecycle-operator/apis/lifecycle/v1alpha1/common/common.go

```diff
@@ -5,179 +5,17 @@ package common
-		return StateUnknown
-	}
-	return StateSucceeded
-}
-
-func TruncateString(s string, max int) string {
-	if len(s) > max {
-		return s[:max]
-	}
-	return s
-}
-
 type CheckType string
 
-const PreDeploymentCheckType CheckType = "pre"
-const PostDeploymentCheckType CheckType = "post"
-const PreDeploymentEvaluationCheckType CheckType = "pre-eval"
-const PostDeploymentEvaluationCheckType CheckType = "post-eval"
-
-type KeptnMeters struct {
-	TaskCount          metric.Int64Counter
-	TaskDuration       metric.Float64Histogram
-	DeploymentCount    metric.Int64Counter
-	DeploymentDuration metric.Float64Histogram
-	AppCount           metric.Int64Counter
-	AppDuration        metric.Float64Histogram
-	EvaluationCount    metric.Int64Counter
-	EvaluationDuration metric.Float64Histogram
-}
-
-const (
-	AppName                 attribute.Key = attribute.Key("keptn.deployment.app.name")
-	AppVersion              attribute.Key = attribute.Key("keptn.deployment.app.version")
-	AppNamespace            attribute.Key = attribute.Key("keptn.deployment.app.namespace")
-	AppStatus               attribute.Key = attribute.Key("keptn.deployment.app.status")
-	AppPreviousVersion      attribute.Key = attribute.Key("keptn.deployment.app.previousversion")
-	WorkloadName            attribute.Key = attribute.Key("keptn.deployment.workload.name")
-	WorkloadVersion         attribute.Key = attribute.Key("keptn.deployment.workload.version")
-	WorkloadPreviousVersion attribute.Key = attribute.Key("keptn.deployment.workload.previousversion")
-	WorkloadNamespace       attribute.Key = attribute.Key("keptn.deployment.workload.namespace")
-	WorkloadStatus          attribute.Key = attribute.Key("keptn.deployment.workload.status")
-	TaskStatus              attribute.Key = attribute.Key("keptn.deployment.task.status")
-	TaskName                attribute.Key = attribute.Key("keptn.deployment.task.name")
-	TaskType                attribute.Key = attribute.Key("keptn.deployment.task.type")
-	EvaluationStatus        attribute.Key = attribute.Key("keptn.deployment.evaluation.status")
-	EvaluationName          attribute.Key = attribute.Key("keptn.deployment.evaluation.name")
-	EvaluationType          attribute.Key = attribute.Key("keptn.deployment.evaluation.type")
-)
-
-func GenerateTaskName(checkType CheckType, taskName string) string {
-	randomId := rand.Intn(99_999-10_000) + 10000
-	return fmt.Sprintf("%s-%s-%d", checkType, TruncateString(taskName, 32), randomId)
-}
-
-func GenerateEvaluationName(checkType CheckType, evalName string) string {
-	randomId := rand.Intn(99_999-10_000) + 10000
-	return fmt.Sprintf("%s-%s-%d", checkType, TruncateString(evalName, 27), randomId)
-}
-
-type GaugeValue struct {
-	Value      int64
-	Attributes []attribute.KeyValue
-}
-
-type GaugeFloatValue struct {
-	Value      float64
-	Attributes []attribute.KeyValue
-}
+// PhaseTraceID is a map storing TraceIDs of OpenTelemetry spans in lifecycle phases
+type PhaseTraceID map[string]propagation.MapCarrier
```

Comment:
Why are we adding this field to old APIs?

Reply:
It was originally placed in `common/phases.go`. Since it was the only type present in the file, I moved it to `common/common.go` (both files are in the same package). Therefore we could remove `common/phases.go` file